inductiva/fluids/scenarios/coastal_area/bathymetry.py

```diff
@@ -264,35 +265,40 @@ def plot(
               the minimum or maximum colors, respectively.
             path: Path to save the plot. If `None`, the plot is not saved, and
               the matplotlib `Axes` object is returned instead.
-            grid_size: Size of the uniform grid to which the bathymetry is
-              interpolated before being plotted.
-            max_distance: Threshold distance to filter out points on the
+            grid_resolution: Resolution, in meters, of the uniform grid to which
```

Comment:
I am having issues understanding how this resolution relates with the two coordinates x,y.

Reply:
The uniform grid in which we plot the bathymetry has a resolution, i.e. a given grid spacing. This sets that resolution.

Reply:
We could, yes. But I thought that setting a single resolution for $x$ and $y$ in this plotting method would suffice. Do you think it is critical to have variable resolution in $x$ and $y$ here?

Reply:
It would make it more uniform with the function .to_uniform_grid(). Wdyt?